content/_code-samples/require-destination-tags/py/require-destination-tags.py

```diff
@@ -0,0 +1,48 @@
+from xrpl.clients import JsonRpcClient
+from xrpl.models.transactions import AccountSet, AccountSetFlag
+from xrpl.transaction import safe_sign_and_autofill_transaction, submit_transaction
+from xrpl.wallet import generate_faucet_wallet
+from xrpl.models.requests import AccountInfo
+
+# Stand-alone code sample for the "Require Destination Tags" tutorial:
+# https://xrpl.org/require-destination-tags.html
+
+lsfRequireDestTag = 131072
+
+# Connect to a testnet node
+print("Connecting to Testnet...")
+JSON_RPC_URL = "https://s.altnet.rippletest.net:51234/"
+client = JsonRpcClient(JSON_RPC_URL)
+
+# Get credentials from the Testnet Faucet
+print("Requesting address from the Testnet faucet...")
+test_wallet = generate_faucet_wallet(client=client)
+myAddr = test_wallet.classic_address
+
+# Construct AccountSet transaction
+tx = AccountSet(
+    account=myAddr,
+    set_flag=AccountSetFlag.ASF_REQUIRE_DEST
+)
+print(f"Prepared transaction: {tx}")
+
+# Sign the transaction
+my_tx_payment_signed = safe_sign_and_autofill_transaction(tx, wallet=test_wallet, client=client)
+print(f"Transaction Hash: {my_tx_payment_signed.txn_signature}")
+
+# Send the transaction to the node
+print(f"Enabling Require Destination Tag flag (asfRequireDest) on {myAddr}")
+submit_transaction = submit_transaction(client=client, transaction=my_tx_payment_signed)
```

Comment:
could you rename the variables here? naming a transaction result the same as an imported function can be confusing

Reply:
For sure.